core-bundle/src/EventListener/PageAccessListener.php

```diff
@@ -76,17 +76,6 @@ private function getPageModel(Request $request): PageModel|null
 
         $pageModel = $request->attributes->get('pageModel');
 
-        if (
```

Comment:
Should this also use the `PageFinder`?

Reply:
I don‘t think so, because `$pageModel` can be an integer here, too. @aschempp /cc

Reply:
I think we discussed this: the request always contains a PageModel. Closing this as resolved.

Reply:
The request always contains a PageModel because of this listener. And judging by the code, `$pageModel` can be an integer in this very method, otherwise `$GLOBALS['objPage']->id === (int) $pageModel` would not make any sense, would it?